deployments/operator/manager/manager.yaml

```diff
@@ -26,13 +26,6 @@ spec:
       - image: intel/intel-deviceplugin-operator:devel
         imagePullPolicy: IfNotPresent
         name: manager
-        resources:
```

Comment:
`manager.yaml` changes need to be in its own PR with a commit message *why* this it is justified. I don't think it's the right fix that we drop these completely.